### DISCUSSION TITLE
Flyby11 0.16

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/FlyOOBE#21] [Announcements]
- The layout of the main UI controls has been improved for better display.
- Some bypass scripts have been optimized.
- Flyby11, in addition to bypassing hardware requirements, can also serve as a handy helper after installation (See Flyby11 Option > Change how you experience Windows). **In such cases, a sleek, fast, and clever app remover (Debloater) is a must, which is now included as [Dumputer](https://github.com/builtbybel/Dumputer).** Simply launch Dumputer.exe to get started. 

### By the way, [Dumputer also has its own repo on GitHub.](https://github.com/builtbybel/Dumputer)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Flyby11/releases/tag/0.16'>Flyby11 0.16</a>.</em>

**Comment by Richard-L** [comment]
Your tools are a blessing, thank you 🙏

**Comment by Owen151243** [comment]
Can you please add a 32 bit version? My laptop is a 32 bit laptop and I can't use this.

**Reply by Belim** [reply]
assembly is available here https://github.com/builtbybel/Flyby11/releases/download/0.16/flybyscript_x86.zip

**Comment by jdai** [comment]
Please help to fix the crash. system info and error pop up attached, please also fix the disply so at least I can view error msg. thx.
![flyby1-120624](https://github.com/user-attachments/assets/e0a02dab-e4cf-4528-8490-6ee65b82cf37)

**Comment by marceli-tar** [comment]
Belim, why does error 502 appear during installation?
![error](https://github.com/user-attachments/assets/3573933a-6248-475f-9b5c-b554e7c251b7)

**Comment by Kurter61** [comment]
there is tpm and secure boot error on self updating PC to 24h2 from 23h2

**Comment by marceli-tar** [comment]
Is there any solution for this? Previously, I updated from Windows 10 to Windows 11 via Rufus

**Comment by jdai** [comment]
I downloaded from MS site the W11, removed sources\appraiserres.dll, then installed/update ok
But, browsers speed are slower, any new link might take 3-5 seconds to show, my RAM is 24G.
I didn't check tpm etc
rgds
joe
________________________________

**Comment by merzouk** [comment]
The script not run.
![Capture d’écran 2024-12-26 235652](https://github.com/user-attachments/assets/10f21db7-0cd4-45ed-bbe4-c46f50eb637b)
